python/uagents-adapter/src/uagents_adapter/__init__.py

```diff
@@ -2,13 +2,31 @@
 
 from importlib import metadata
 
-from .a2a_outbound import A2AAgentConfig, MultiA2AAdapter, SingleA2AAdapter, a2a_servers
-from .a2a_inbound import A2ARegisterTool
 from .common import ResponseMessage, cleanup_all_uagents, cleanup_uagent
 from .crewai import CrewaiRegisterTool
 from .langchain import LangchainRegisterTool
 from .mcp import MCPServerAdapter
 
+# Conditional imports for A2A modules
+try:
+    from .a2a_inbound import A2ARegisterTool
+    from .a2a_outbound import (
+        A2AAgentConfig,
+        MultiA2AAdapter,
+        SingleA2AAdapter,
+        a2a_servers,
+    )
+
+    A2A_AVAILABLE = True
+except ImportError:
+    A2A_AVAILABLE = False
+    # Create dummy classes for when A2A is not available
+    A2ARegisterTool = None
+    A2AAgentConfig = None
+    MultiA2AAdapter = None
+    SingleA2AAdapter = None
+    a2a_servers = None
```

Comment:
Is this really needed? I think it would be better to raise an import error if these are not installed.